examples/mobile/HomeApp/js/app.js

```diff
@@ -104,6 +104,11 @@
 		activeWebClipList.forEach(function (webclip) {
 			var card = document.createElement("div");
 
+			// check directory in weblclip
+			if (!webclip.match(/\.html$/) && !webclip.match(/\/$/)) {
```

Comment:
No other cases exist?
ex) htm, xhtml

Reply:
if webclip directory contains files webcliplist API (/api/webcliplist) will return them too
so we need to modify server API or make changeas @DohyungLim pointed

Reply:
By default the `index.html` is start point for web apps but this can be changed by developer eg. in `config.xml` for tizen web apps. This is solution for the moment. I can add next regex expression if you wish.

Reply:
I'll remove matching for file extension. The webclip list contains only directories of webclips.

Reply:
ok. good.